### ISSUE TITLE
VISUAL_CONFIG for Epoch20_LXRT.pth

### ISSUE THREAD

**Issue by ecekt** [airsplay/lxmert#59]
Hello, I am trying to extract the cross-modal representations from Epoch20_LXRT.pth. I am using entry.py. What should be the args for l_layers, x_layers and r_layers for this model? I tried various values and it says some weights are missing in either loaded or model.

Best regards.

**Comment by airsplay** [comment]
It is because the pre-trained model has more weights in predicting the masked values, which will not be loaded in fine-tuning / extraction.

If you load BERT weights for other tasks, you would get the same message.

**Comment by ecekt** [comment]
Ok, I understand. But then what should I input as the layer values? 9,5,5?
